### PR TITLE
Get rid of jq

### LINKED ISSUE #5: ./launch-s0-instance.sh fails on amazon 2
It fails with an error
```
[elastio@amazon2-amd64-test scripts]$ ./launch-s0-instance.sh -s 123 -k rinastestkms -b rinastestbucket -p admin
Validating bucket "rinastestbucket"...
Validating KMS key alias "rinastestkms"...
./launch-s0-instance.sh: line 306: jq: command not found
The AWS CLI is configured to use the 'eu-central-1' region. And an ec2 instance will be launched in this region.
However the KMS key 'rinastestkms' is located in the different region ''.
Please chouse another KMS key in the 'eu-central-1' region or change current region to the ''.
NOTE: The launched ec2 instance, s3 bucket and KMS key should be in the same region!
```

Looks like the util `jq` is not installed, so, the script should handle it and install it if it's not already installed

### PR DESCRIPTION
**Pull request by e-kov**
Resolves #5 in another way.

We aren't going to install `jq` because it's too strange to install something from the script directly.
The output and `sudo` password request seems to be strange.
But from @mvovchuk words, an error message about not installed `jq` and instruction how to install it seems to be toooo difficult for the potential customers... That's why just don't use `jq` at all. `grep`, `tr` and `awk` are our friends.

### CODE REVIEW

**Comment by JusSergey** [comment]
@e-kov are `tr` and `awk` installed by default on all Linux distros? if yes then it's cool) if not then we still will get errors like `tr: command not found` or `awk: command not found`

File: scripts/launch-s0-instance.sh [hunk 1]
```diff
@@ -237,27 +237,6 @@ if ! which aws >/dev/null 2>&1 || [[ $(aws --version | cut -d' ' -f1 | cut -d'/'
     exit 1
 fi
 
-# Try to install 'jq' if it's not existing or fail
-if ! which jq; then
```
Comment (JusSergey) [inline scripts/launch-s0-instance.sh:241]: @e-kov are tr and awk installed by default on all Linux distros? if yes then it's cool) if not then we still will get errors like tr: command not found or awk: command not found

Comment (e-kov) [reply scripts/launch-s0-instance.sh:241]: yes, they are always present even in the minimal installs of the supported distros.

**Review by JusSergey — APPROVED** [review]

**Comment by anelson** [comment]
Haha.  This is why we can't have nice things )

**Review by anelson — APPROVED** [review]

**Pull request merged**